Documentation/gettingstarted.rst

```diff
@@ -1013,33 +1016,43 @@ The output should simply return:
     OK
 
 
-Run a script to create the client requests to the newly created web-server. The script should generate a ``client.json`` for the client task:
+Run a script to create "good client" and "bad client" requests to the newly created web-server. The script should generate ``goodclient.json`` and ``badclient.json`` files for the client tasks, respectively:
```

Comment:
Thanks- I added an argument check in generate_client_task.sh and tail_client.sh.